### DISCUSSION TITLE
Combine Different Images to One

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7335] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

I have a regular 1920x1080 image, and a 1972x1421 photo stitch. How do I combine them together without getting white spaces like this?

![tttt](https://github.com/ImageMagick/ImageMagick/assets/141977822/dcf2678e-f5e4-41cc-9e6f-d173294abec1)

**Comment by dlemstra** [comment]
Please attach your images and the command that you are executing.

**Reply by Jay123210599** [reply]
Images: https://imgur.com/a/WzamEvX 
Here's the command: ``magick image.png image_stitch.png -append output.png``

**Comment by fmw42** [comment]
Works fine for me on IM 7.1.1.32 Mac OSX Ventura, since both images have the same width.

`magick img1.webp img2.webp -append x.png`

![x](https://github.com/ImageMagick/ImageMagick/assets/13458218/5d2384a3-2bf0-4f47-8936-21269cf92a1a)


If you have other images that do not work (not the same width), then post them, so we can test with them.

**Reply by Jay123210599** [reply]
Well, it didn't work for me, and I'm on Windows. What should I do?
![image](https://github.com/ImageMagick/ImageMagick/assets/141977822/30a4df67-431f-4ed9-aba0-50d861a0dccb)
![0000_stitch](https://github.com/ImageMagick/ImageMagick/assets/141977822/6d7bc57a-b0c0-497d-a84e-840686ea26c1)

**Reply by fmw42** [reply]
Sorry, I got low resolution versions that were the same width.  These are high resolution with different widths.  So what do you want to put in that space?  Do you want it to be transparent or do you want it be some specific color or do you want to enlarge the smaller width image or do you want to shrink the larger image, so that both would have the same width and thus match?

If you want it transparent, then do

`magick img1.png img2.png -background none -append x.png`

or if you want to make the smaller width image larger, then 

`magick img1.png img2.png -resize "%[fx:max(u.w,v.w)]x" -append y.png`

![y](https://github.com/ImageMagick/ImageMagick/assets/13458218/4ae669ab-7904-4d00-a170-28b4cc3a0334)
